docs/developer/running-saleor/configuration.mdx

```diff
@@ -173,6 +173,12 @@ Controls whether registering new customer accounts should require email confirma
 
 Defaults to `True`.
 
+### `GRAPHQL_QUERY_MAX_COMPLEXITY`
+
+Controls maximum allowed complexity (cost) of GraphQL query. If query's complexity exceeds this limit, it will not be executed and GraphQL API will return an error instead.
```

Comment:
Would like to know more about how do we define query cost.

Reply:
@krzysztofwolski I've discussed this with @maarcingebala and instruction for defining costs was added as module doc to python module defining cost map.